tests/config.go

```diff
@@ -37,6 +37,10 @@ type Config struct {
 	CertFile         string
 	KeyFile          string
 
+	PDMaxReplicas       int `yaml:"pd_max_replicas" json:"pd_max_replicas"`
```

Comment:
I think it doesn't need these flags, just set a literal value to https://github.com/pingcap/tidb-operator/pull/479/files#diff-f2f48876f13bab9311a47b1c1be4a1f9R207 is enough.

```
		clusterInfo = clusterInfo.
			UpdatePdMaxReplicas(5).
			UpdateTiKVGrpcConcurrency(1024).
			UpdateTiDBTokenLimit(8)
```

Reply:
Yes, using literal value is simple and clear. Maybe I don't have to do this at the very beginning, but the motivation is that I want these parameters being configurable in favor of flexibility, I tend to keep it because it does no harm even if we won't configure it in the near future. How do you think?

Reply:
I am fine with this solution.